src/components/Login/Login.tsx

```diff
@@ -42,24 +43,37 @@ const Login = () => {
       phone: phoneInputRef.current?.value.trim(),
       password: passwordInputRef.current?.value.trim(),
     }
-    //https://semicolon-registration-backend.onrender.com/auth/login
-    fetch('https://semicolon-registration-backend.onrender.com/auth/login', {
-      method: 'post',
-      body: JSON.stringify(body),
-      headers: {
-        'Content-type': 'application/json; charset=UTF-8',
-      },
-    })
+    // //https://semicolon-registration-backend.onrender.com/auth/login
+    // fetch('https://semicolon-registration-backend.onrender.com/auth/login', {
+    //   method: 'post',
+    //   body: JSON.stringify(body),
+    //   headers: {
+    //     'Content-type': 'application/json; charset=UTF-8',
+    //   },
+    // })
+    //   .then((response) => {
+    //     if (response.status === 401) {
+    //       throw new Error('Page not found')
+    //     }
+    //     if (response.status === 200) {
+    //       // throw new Error('Accepted')
+    //       navigate('/stats')
+    //       // return redirect('/stats')
+    //     }
+    //     return response.json()
+    //   })
+    //   .catch((err) => console.log(err))
+    axios
+      .post(
+        'https://semicolon-registration-backend.onrender.com/auth/login',
+        body
+      )
       .then((response) => {
-        if (response.status === 401) {
-          throw new Error('Page not found')
+        if (response.data.status == 'success') {
+          localStorage.setItem('user', JSON.stringify(response.data.data))
```

Comment:
refractor: create a function in auth for this. Convert `authHeader.ts` into a general `auth.ts`